client/src/components/AdminUI/AdminUI.js

```diff
@@ -1,20 +1,109 @@
-import React from "react";
 import { useState, useEffect } from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
-import { Checkbox, Container, Typography, Button, List, ListItem, ListItemText, TextField } from '@mui/material';
+import { Container, Typography, Button } from '@mui/material';
+import { DataGrid } from '@mui/x-data-grid';
 import httpClient from '../../httpClient';
 
 function AdminUI() {
+  const [isAdmin, setIsAdmin] = useState(false)
   const { isAuthenticated, user, loginWithRedirect } = useAuth0();
-  const isAdmin = user?.["https://girard-server.herokuapp.com/roles"]?.includes('admin');
   const [users, setUsers] = useState([]);
   useEffect(() => {
-      if (isAuthenticated && isAdmin) {
-          httpClient.get(process.env.REACT_APP_API_SERVER + "/users")
-              .then(res => setUsers(res.data))
-      }
-  }, [isAuthenticated, isAdmin])
-    if (!(isAuthenticated) || !(isAdmin)) {
+        setIsAdmin(user?.["https://girard-server.herokuapp.com/roles"]?.includes('admin'));
```

Comment:
Any reason why the URL is in here?

Reply:
I used the URL as a namespace for the role information on the JWT. This seems to be the convention, but I think it could be any other string you might want. This URL value is never actually used by anything so far as a web address.